include/deal.II/matrix_free/shape_info.h

```diff
@@ -102,7 +102,12 @@ namespace internal
       /**
        * Shape functions without an tensor product properties.
        */
-      tensor_none = 6
+      tensor_none = 6,
+
+      /**
+       * Shape functions without an tensor product properties.
+       */
+      tensor_raviart_thomas = 7
```

Comment:
Can you move this before `tensor_none`? It shouldn't make a difference with either value, but it feels most logical to have the case without tensor products (i.e., triangles, tetrahedra, pyramids, wedges) as the last option, whereas everything with some tensor product structure belongs together.

Reply:
I agree! I had seen that in some places `<` or `>` is used, so to avoid anything weird I just placed it last. But I guess since `tensor_none` is last, `<` or `>` wouldn't make sense anyway :)